.github/workflows/finalize-release.yaml

```diff
@@ -27,6 +27,12 @@ jobs:
         run: |
           git config --local user.email 38288322+imodeljs-admin@users.noreply.github.com
           git config --local user.name imodeljs-admin
+
+      - name: Setup node
+        uses: actions/setup-node@v2
```

Comment:
please update to the use the latest version of these gh actions, 
https://github.com/actions/setup-node/releases

Reply:
please update the other actions and any other gh workflow, 
eg actions/checkout@v2